source/common/config/api_type_oracle.cc

```diff
@@ -1,8 +1,5 @@
 #include "common/config/api_type_oracle.h"
 
-#include "common/common/assert.h"
-#include "common/common/logger.h"
```

Comment:
Do you mind cleaning up the BUILD file also? https://github.com/envoyproxy/envoy/blob/d1db56424d39f10715f81d17d80079a22040abbf/source/common/config/BUILD#L16-L17

Reply:
good call, updated

Reply:
assert is still used here, but removed logger: 

https://github.com/envoyproxy/envoy/blob/d1db56424d39f10715f81d17d80079a22040abbf/source/common/config/version_converter.cc#L5

Reply:
looks like `version_converter_lib` was transitively getting `assert_lib` by depending on `api_type_oracle_lib` even though it uses `assert_lib` directly. fixed the dependencies in the BUILD file accordingly